go.mod

```
@@ -256,7 +256,7 @@ require (
 )
 
 replace (
-	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.28.0-sdk-v0.46.16
+	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.28.0-sdk-v0.46.16.0.20250331072412-0673d7c9b833
```

Just created a release for you: https://github.com/celestiaorg/cosmos-sdk/releases/tag/v1.28.1-sdk-v0.46.16

```suggestion
	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.28.1-sdk-v0.46.16
```